### ISSUE TITLE
System asks permission for Accessibility in Privacy settings again after updating the application

### ISSUE THREAD

**Issue by niw** [niw/HapticKey#22]
When updating the application using “Check for updates…” menu (by using Sparkle,) after updating the application binary, system asks permission for Accessibility in Privacy settings again, even if it’s previously approved and the application appears in the list with a checkmark.

For short-term solution, to workaround this behavior, toggle the checkmark and enable HapticKey again from the menu item.

**Comment by niw** [comment]
To workaround this behavior, seems like, it requires a code signing with developer ID.
This cost some on me and not trivial for now. I will figure out it but it may not be immediate.

**Comment by rodrigoAbril** [comment]
Im my case I cant toggle the permission even after using the admin password, the checkbox is perpetually ON but the application says the permission for Accessibility  is missing...  any Ideas?

**Comment by niw** [comment]
@rodrigoAbril What is your macOS version? Also wondering if your mac has any MDM profiles (such as company issued laptop.) If you can, select app and use minus button at bottom, then restart app again that will create new entry then I think you can enable checkmark again.
